src/lib/notifications/Snackbar.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import Icon from '$lib/component/Icon.svelte';
+  import Icons from '$lib/component/Icons';
+  import { slide, fly } from 'svelte/transition';
+  import type { Notification } from '.';
+
+  export let notification: Notification;
+</script>
+
+<div
+  out:slide
+  in:fly={{ x: 100 }}
+  class="mb-4 flex h-auto w-80 rounded-lg bg-primary px-4 py-2"
+  role="alert"
```

Comment:
Maybe we shouldn't use `role="alert"` for every single snackbar notification, but rather `role="status"` for the majority of them.

Here's what I've read on the subject:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role
"The alert role is used to communicate an important and usually time-sensitive message to the user."
"Warning: Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role
"A status is a type of [live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) providing advisory information that is not important enough to justify an alert, which would immediately interrupt the announcement of a user's current activity."

If we aren't using notifications to only display very important information, then the status role is a more adequate default value, as it will not cause screen readers to interrupt their current activity.
In my opinion, leaving it with `role="status"` should work for now. In the future, adding a more urgent variation of a notification which uses the alert role and no timeout (or a longer one) could be an option. 

What do you think?

Reply:
I definitely agree with you! I wasn't aware of that difference, I'll make the change. Thanks!